python-frontend/src/test/java/org/sonar/plugins/python/api/cfg/ControlFlowGraphTest.java

```diff
@@ -694,6 +694,16 @@ void class_def() {
     );
   }
 
+  @Test
+  void class_def_elements() {
+    ControlFlowGraph cfg = cfg(
+      "class Dec:",
+      "    ...");
+    List<Tree> elements = cfg.start().elements();
+    assertThat(elements).hasSize(2);
+    assertThat(elements).extracting(Tree::getKind).containsExactly(Kind.CLASSDEF, Kind.EXPRESSION_STMT);
+  }
```

Comment:
This is an extra test to enforce the first element of the CFG to be a `CLASSDEF`.